src/Optimizely/Entity/Attribute.php

```diff
@@ -36,6 +46,14 @@ public function __construct($id = null, $key = null)
         $this->_key = $key;
     }
 
+    /**
+     * @return boolean Representing whether attributes are valid or not.
+     */
+    public static function getConstants()
```

Comment:
Don't edit the Attribute class file at all. 
Supported attributes are in context to creating payload. Attribute key by definition can hold any value